src/pocketmine/command/defaults/TimingsCommand.php

```diff
@@ -123,17 +135,12 @@ public function onCompletion(Server $server){
 							$server->getLogger()->logException($result);
 							return;
 						}
-						list(, $headers) = $result;
-						foreach($headers as $headerGroup){
-							if(isset($headerGroup["location"]) and preg_match('#^http://paste\\.ubuntu\\.com/([A-Za-z0-9+\/=]+)/#', trim($headerGroup["location"]), $match)){
-								$pasteId = $match[1];
-								break;
-							}
+						if(isset($result[0])){
+							$pasteId = json_decode($result[0])->id ?? 0;
```

Comment:
and if `json_decode()` returns `false`, what then?

Reply:
@dktapps try this code
```php
$obj = (object) false;
var_dump($obj);
var_dump($obj->id ?? 0);
```

Reply:
your error handling should be explicit... and the paste ID should not become zero otherwise a broken link will be reported.